packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.schema.cue

```diff
@@ -0,0 +1,792 @@
+package dashboard
+
+DashboardV2: {
```

Comment:
@ivanortegaalba  - i don't think we gonna need this type. This envelope is going to be provided by the API. What's relevant in this schema is the dashoard spec.